packages/server-wallet/e2e-testing/scripts/load-generator.ts

```diff
@@ -43,30 +47,52 @@ async function createLoad() {
       default: 'temp/test_load.json',
     })
     .option('roleFile', {
-      alias: 'f',
+      alias: 'rf',
       describe: 'The path to a file containing the role information.',
       default: './e2e-testing/test-data/roles.json',
     })
+    .option('fundingStrategy', {
+      alias: 'f',
+      describe: 'Whether channels are funded directly or by ledger channels.',
+      choices: ['Ledger', 'Direct'],
+      demandOption: true,
+    })
     .option('duration', {
       alias: 'd',
       min: 10,
-      default: 30,
+      default: 60,
       describe: `The amount of time (in seconds) that the load should run for.
       This dictactes the max timestamp a step can have.`,
     })
     .option('createRate', {
       alias: 'cr',
       min: 1,
-      default: 5,
+      default: 1,
       describe: 'The number of channels that should be created per a second.',
     })
-    .option('createWait', {
+    .option('createLedgerDuration', {
```

Comment:
There are starting to be a lot of options which is going to make the load generator hard to use. We probably want to simplify them? Would appreciate any thoughts on how to do that!

Reply:
Could you allow the options to be defined in a referenced json file? Something like the way `tsc` will accept command line args , or you can point it at a `tsconfig.json` file. 

I dug into `yargs` docs and found this https://github.com/yargs/yargs/blob/master/docs/advanced.md#rc-files .

Reply:
That's a good option!